test/ret/discord_client_test.exs

```diff
@@ -65,4 +67,44 @@ defmodule Ret.DiscordClientTest do
   test "moderator should be able to view restricted channel" do
     assert @moderator_user_id |> DiscordClient.has_permission?(@restricted_channel_binding, :view_channel)
   end
+
+  @tag only: true
+  test "should make a properly structured http request" do
```

Comment:
May want an additional test for a request that _should_ add the user agent, if we have a specific example.